### ISSUE TITLE
Parsing failure for JVM bytecode generated from Scala

### ISSUE THREAD

**Issue by rrwright** [GaloisInc/jvm-parser#2]
Trivial Scala code which compiles to only simple Java works. Slightly more interesting Scala which depends on the `scala-library` for collections fails with a parse error:
```
saw: Index 587 is not a method reference.
CallStack (from HasCallStack):
  error, called at src/Language/JVM/Parser.hs:387:12 in jvm-parser-0.3.0.0-8vikjDDT9aZHOxrZiDNeeN:Language.JVM.Parser
```

<img width="1680" alt="screen shot 2019-02-05 at 11 37 11 am" src="https://user-images.githubusercontent.com/680995/52313075-f9bd2580-2961-11e9-917a-ef5ed4a4c58c.png">

This example was produced with the nightly build of SAW: `saw-0.2-2019-01-29-MacOSX-64`

**Comment by brianhuffman** [comment]
I suspect this is the same issue as GaloisInc/saw-script#920. It might also be fixed by #5.

**Comment by brianhuffman** [comment]
I'm closing, on the assumption that #5 fixed the problem. Please reopen if the example from the original post still causes a crash.
